make/modules/java.desktop/Lib.gmk

```diff
@@ -93,6 +93,7 @@ ifeq ($(call isTargetOs, macosx), true)
       LDFLAGS := $(LDFLAGS_JDKLIB) \
           $(call SET_SHARED_LIBRARY_ORIGIN), \
       LIBS := \
+          -ljava \
```

Comment:
There should be a dependency declaration added too. Something like this right after the SetupJdkLibrary macro call:

$(BUILD_LIBOSXAPP): $(call FindLib, java.base, java)

Reply:
> There should be a dependency declaration added too. Something like this right after the SetupJdkLibrary macro call:
> 
> $(BUILD_LIBOSXAPP): $(call FindLib, java.base, java)

Fixed.